### DISCUSSION TITLE
Remote development for Linux with GLIBC < 2.29?

### DISCUSSION THREAD

**Discussion by LuckyDC** [zed-industries/zed#15162] [Feature Requests]
On Ubuntu18.04, GLIBC is 2.27 which does not satisfy the requirement of zed remote development. However, there are still so many servers running on Ubuntu18.04. VSCode also support a workaround for remote development on platforms with older GLIBC.

**Comment by jandrej** [comment]
I tried using `LD_LIBRARY_PATH` to inject a more recent version of glibc and manually start the remote server without success, too. Is there a way of building the server manually or is the regular zed binary just being started in server mode?

**Comment by sirynka** [comment]
Would it be possible to compile zed with statically linked glibc - musl?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
